www/css/cytube.css

```diff
@@ -173,24 +155,14 @@
     overflow-y: scroll;
 }
 
-#plmeta {
-    border: 1px solid #aaaaaa;
-}
+
 
 #pllength {
     float: right;
 }
 
-.queue_entry {
```

Comment:
This has been resolved and the default properties have been returned to cytube.css